PREREQUISITE.md

```diff
@@ -0,0 +1,263 @@
+>
+> > If you want to know more about 'Gradle', [https://www.baeldung.com/gradle-build-settings-properties](https://www.baeldung.com/gradle-build-settings-properties) will be helpful
+>
+> <img width="392" alt="module_007" src="https://user-images.githubusercontent.com/27720475/130185391-02c82964-299e-484c-80e1-810254dd7070.png">
+>
+> > If Caused by: groovy.lang.MissingPropertyException: Cannot get property 'signing.keyId' on extra properties extension as it does not exist error occur, please delete the sentence below in euphony/gradle.build and click 'Sync Now'apply from: file('publish.gradle')
+>
+> <img width="392" alt="module_008" src="https://user-images.githubusercontent.com/27720475/130185404-16a862f0-28ee-4c0e-917c-826fe73d25bb.png">
+> </details>
+
+4. Now, we'll gonna add a dependency. In `File` > `Project Structure`, click `+` button at `Dependencies` > `app`. Then click `Module Dependency`
+
+<img width="392" alt="module_009" src="https://user-images.githubusercontent.com/27720475/130185413-7dc68c68-c756-455a-8829-8d61e128b939.png">
+
+<img width="392" alt="module_010" src="https://user-images.githubusercontent.com/27720475/130185419-68d5df4c-7f71-4d08-b1dd-bea9ddd975c2.png">
+
+After that, you can see a sentence is added in `build.gradle(Module:app)`
+
+```gradle
+dependencies{
+     ...
+      implementation project(path: ':euphony')
+     ...
+}
+
+```
+
+> Path could be different, if you change module name at step 3
+
+---
+
+## 2. Edit your AndroidManifest.xml
+
+Add the following line to the AndroidManifest.xml:
+
+```xml
+<uses-permission android:name="android.permission.RECORD_AUDIO" />
+```
+
+### Request permission to users
+
+Plus, you should consider that the permission model is different depending on the user's android version.
+Its details is below.
+
+<details>
+    <summary>Show details</summary>
+
+So, if you wanna use this library in the right way, you should check your permission manually in app configuration. That's because after Android 6.0 Marshmallow, Android introduced a new permissions model that lets apps request permissions from the user at runtime, rather than prior to installation.
+
+You can read a below docs to check details.[https://developer.android.com/training/permissions/usage-notes?hl=en](https://developer.android.com/training/permissions/usage-notes?hl=en)
+
+So when you develop the android app, you should consider when app requests permission to users, and you should typing codes that requests permssions to users.
+
+Over the all, as this library's minimun sdk is 14, we must consider two types app 14<= sdk < 16 app and sdk >= 16 app.
+
+Let's check out how can it possible.
+
+Because my test app's sdk is 30, i should consider about it.
+
+I made button to request user permissions. So, if you push that button, request alert pops up, and if you denied permission, app is closed automatically.
```

Comment:
I think that the expressions of `I`, `my` are careful to indicate it in the official document.
So, how about  below version?

```suggestion
The sample code below is an example of adding permission through the UI. Please refer to it.
```